summerofcode/2023.md

```diff
@@ -26,12 +26,26 @@ If you are a project maintainer and consider mentoring during the GSoC 2023 cycl
 
 ## Ideas
 
+- [DevStream](#devstream)
+  - [Developer Tool Improvement](#developer-tool-improvement)
 - [Knative](#knative)
   - [Multiple Knative Eventing control planes](#multiple-knative-eventing-control-planes)
   - [Eventing Sender Identity](#eventing-sender-identity)
   - [NetworkPolicy support for Knative Serving](#networkpolicy-support-for-knative-serving)
   - [Improving Observability in Knative Eventing](#improving-observability-in-knative-eventing)
 
+### DevStream
+
+#### Developer Tool Improvement
+
+- Description: DevStream wants to build a developer tool that can bootstrap your local dev env at the click of a button using interactive CLI. Currently we are doing MVP and market researches. We want to add more support regarding more programming languages and frameworks and CI/CD tools so that the users can bootstrap a new microservice using predefined template and setup the whole CI/CD pipelines locally.
+- Expected outcome: creating PRs to support more languages, frameworks, and tools.
+- Recommended Skills: Golang, understanding of popular programming languages and mainstream frameworks, knowledge of famous DevOps tools such as GitHub Actions, GitLab CI, FluxCD, etc.
+- Mentor(s): Tiexin Guo @ironcore864 guotiexin@gmail.com, Daniel Hu @daniel-hutao tao.hu@merico.dev
```

Comment:
Clarified that mentors are maintainers: https://github.com/cncf/foundation/blob/main/project-maintainers.csv#L1091